docs/concepts/collections.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 1
+---
+
+# Collections
+
+Collections are a connecting link between blocks and transactions. They make an ordered list of one or more hashes of signed transactions. These are formed by collection node clusters. Why blocks don’t directly include transactions (as they do on Ethereum) is due to the optimization of data that needs to be transferred on the network. Consensus nodes need to only agree on the order of transactions to be executed, they don’t need to know the transaction payload, thus making blocks and collections lightweight. Collection nodes hold transaction payloads for anyone that requests them (e.g. execution nodes). 
+
+![Screenshot 2023-08-17 at 19.50.39.png](_collection_images/Screenshot_2023-08-17_at_19.50.39.png)
+
+## Collection Retrieval
+
+You can use the Flow CLI to get the collection data by running:
+
+```
+flow collections get caff1a7f4a85534e69badcda59b73428a6824ef8103f09cb9eaeaa216c7d7d3f -n mainnet
+```
+
+Find [more about the command in the CLI docs](https://developers.flow.com/next/tools/toolchains/flow-cli/get-flow-data/get-collections).
```

Comment:
use relative links directly to md/mdx files

Reply:
Good flag